helpdesk_factorlibre/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,203 @@
+ # -*- coding: utf-8 -*-
+# © 2020 Nacho Morales
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import models, fields, _, api
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class HelpdeskTicket(models.Model):
+    _name = 'helpdesk.ticket'
+    _inherit = ['mail.thread', 'mail.activity.mixin']
+    
+    #Aqui se meten los metodos Default que vaya a utilizar, antes de los campos de la clase
+    
+    def _get_default_priority(self):
+        return "1"
+    
+    def _get_default_stage_id(self):
+        return self.env['helpdesk.ticket.stage'].search([], limit=1).id
+    
+    def _get_default_team_id(self):
+        return self.env['helpdesk.ticket.team'].search([], limit=1).id
+    
+    def _get_default_user_id(self):
+        return 1
+    
+    
+    ticket_number = fields.Integer(
+        string='Numero de ticket',
+        required=True,
+    )
+    
+    name = fields.Char(
+        string='Tittle',
+        required=True,
+    )
+    
+    description = fields.Html(
+        string='Description',
+    )
+    
+    assigned_date = fields.Datetime(
+        string='Assigned Date',
+        compute='_compute_assigned_date',
+        store=True,
+    )
+    
+    closed_date = fields.Datetime(string='Closed Date')
+    
+    priority = fields.Selection(selection=[
+        ('0', _('Baja')),
```

Comment:
avoid using Spanish in the code

Reply:
Fix in 31c0c3d9567a5b7680d44351437cb1eacb729ef6